CONTRIBUTING.md

```diff
@@ -71,29 +71,29 @@ If you can code, see also how to [develop your own quest](#developing-new-quests
 ## Improving documentation
 
 You can also help to keep the OpenStreetMap wiki **up-to-date** for StreetComplete. For example:
-* Add missing quests in [the quest list](https://wiki.openstreetmap.org/wiki/StreetComplete/Quests) or check, that they are up to date.
+* Add missing quests in [the quest list](https://wiki.openstreetmap.org/wiki/StreetComplete/Quests) or check, that they are up to date; there is a [helper action](https://github.com/streetcomplete/StreetComplete/actions/workflows/generate-quest-list.yml) which generates a CSV for this.
 * Edit [the FAQ](https://wiki.openstreetmap.org/wiki/StreetComplete/FAQ) and add reoccurring questions.
 * Edit [the main StreetComplete page](https://wiki.openstreetmap.org/wiki/StreetComplete).
-* Edit [the JSON of all tags used by StreetComplete](https://github.com/goldfndr/StreetCompleteJSON) that is displayed on [taginfo](https://taginfo.openstreetmap.org/projects/streetcomplete).
+* Edit [the JSON of all tags used by StreetComplete](https://github.com/goldfndr/StreetCompleteJSON) that is displayed on [taginfo](https://taginfo.openstreetmap.org/projects/streetcomplete); there is a [draft Pull Request](https://github.com/streetcomplete/StreetComplete/pull/2754) to automate this).
```

Comment:
I am not sure is it useful to link draft PRs - it will become stale soon either way (either it will be closed unfinished or become implemented)

Reply:
Yes, I pondered about it, but I'd feel bad if someone spent a load of time manually crafting JSON to update the existing stuff by hand in the time between this getting merged and the automatic generation getting merged, when this PR has already done that processing.

I'd hope worst case even if it wasn't deemed practical to generate the full replacement JSON, it would remain as a service a bit like the CSV thing, to simplify manual updates, but that's probably a discussion for #2754 ...